src/components/postgresConnect.ts

```diff
@@ -123,10 +123,14 @@ class PostgresDB {
 
   public getCounter = async (counterName: string, increment = true) => {
     const textSelect = 'SELECT value FROM counter WHERE name = $1;'
+    const textNewCounter = 'INSERT INTO counter (name, value) VALUES($1, 1)'
     const textUpdate = 'UPDATE counter SET value = value + 1 WHERE name = $1;'
     try {
       await this.query({ text: 'BEGIN TRANSACTION;' })
       const result: any = await this.query({ text: textSelect, values: [counterName] })
+      if (result.rows.length === 0) {
+        await this.query({ text: textNewCounter, values: [counterName] })
+      }
```

Comment:
I was going to use the counter for creating a trigger event id, but it didn't end up working for database constraint reasons. 

However, this seems like a sensible improvement and probably should have been there all along, so I've left it in -- if a counter doesn't exist yet, it'll get created the first time it's queried. (So when creating an action that uses a new counter, we don't have to manually create the database record for it to work)